fili-core/src/main/java/com/yahoo/bard/webservice/jobs/AsyncUtils.java

```diff
@@ -46,7 +47,7 @@ public static Response makeJobMetadataResponse(String jobMetadata) {
      * @return A PreResponse containing the error information that should be sent to the user
      */
     public static PreResponse buildErrorPreResponse(Throwable throwable) {
-        ResponseContext responseContext = new ResponseContext(Collections.emptyMap());
+        ResponseContext responseContext = new ResponseContext(new LinkedHashMap<>());
```

Comment:
That sounds dangerous to me, not unless we take steps to make sure the empty map is immutable. The reason `Collections.emptyMap` is safe is because it's immutable, so you don't have to worry about someone accidentally modifying the map and spreading the changes all over the system.

Frankly, the overhead of building an empty collection is _so tiny_ that I doubt creating our own custom empty, immutable `LinkedHashMap`is not worth the effort.